doc/radosgw/layout.rst

```diff
@@ -119,7 +125,7 @@ is controlled by a 'policy' setting.[3]
 An RGW object may comprise multiple RADOS objects, the first of which
 is the ``HEAD`` that contains metadata including manifest, ACLs, content type,
 ETag, and user-defined metadata. The metadata is stored in xattrs.
-The ``HEAD` object may also inline up to :confval:`rgw_max_chunk_size` of object data, for efficiency
+The ``HEAD`` object may also inline up to :confval:`rgw_max_chunk_size` of object data, for efficiency
```

Comment:
I think we're going to need a definition of "inline" for the glossary.

Reply:
https://docs.ceph.com/en/latest/cephfs/experimental-features/#inline-data lists that this feature would "likely" be removed in Quincy.

I am not familiar with this feature but this link seems to explain it pretty well.  
If this describes the same functionality then I believe `experimental-features.rst` needs to be updated to either state that the feature was removed (when), remove the statement altogether, or update to talk about a different release where it might be removed if the decision has not yet been taken.

I can take a look if I find some commits that prove this either way.

Reply:
And the code is still the same in `main`. I will prepare a PR to update https://docs.ceph.com/en/latest/cephfs/experimental-features/#inline-data for `s/the Q release/a future release/`.

Reply:
PR done in #63918. Anyways, the RGW feature that's being discussed might have nothing to do with the CephFS feature deprecation. Both probably use the same RADOS feature.